StudioCore/Resource/ResourceManager.cs

```diff
@@ -253,7 +253,15 @@ private static void LoadBinderResources(LoadBinderResourcesAction action)
 
                 foreach (var t in action.PendingTPFs)
                 {
-                    var f = TPF.Read(action.Binder.ReadFile(t.Item2));
+                    TPF f = null;
+                    try
+                    {
+                        f = TPF.Read(action.Binder.ReadFile(t.Item2));
+                    }
+                    catch 
+                    {
+                        continue; 
+                    }
                     action._job.AddLoadTPFResources(new LoadTPFResourcesAction(action._job, t.Item1, f, action.AccessLevel, ResourceManager.Locator.Type));
```

Comment:
You can move the TPF declaration and the AddLoadTPFResources call under the try statement and eliminate the continue inside the catch.